src/sage/coding/binary_code.pyx

```diff
@@ -1279,25 +1279,33 @@ cdef class OrbitPartition:
         nwords = (1 << nrows)
         self.nwords = nwords
         self.ncols = ncols
-        self.wd_parent =        <int *> sig_malloc( nwords * sizeof(int) )
-        self.wd_rank =          <int *> sig_malloc( nwords * sizeof(int) )
-        self.wd_min_cell_rep =  <int *> sig_malloc( nwords * sizeof(int) )
-        self.wd_size =          <int *> sig_malloc( nwords * sizeof(int) )
-        self.col_parent =       <int *> sig_malloc( ncols * sizeof(int) )
-        self.col_rank =         <int *> sig_malloc( ncols * sizeof(int) )
-        self.col_min_cell_rep = <int *> sig_malloc( ncols * sizeof(int) )
-        self.col_size =         <int *> sig_malloc( ncols * sizeof(int) )
+        self.wd_parent = <int *> sig_malloc(nwords * sizeof(int))
```

Comment:
Why not using the `MemoryAllocator` to simplify the code ?

Reply:
certainly a good idea, but not for here ?

Reply:
sure, to be added to the long todo list.